2023/f8995.py

```diff
@@ -0,0 +1,30 @@
+from form import Form, FilingStatus
+
+class F8995(Form):
+    """Form 8995A, Qualified Business Income Deduction Simplified Computation"""
+    def __init__(f, inputs, f1040, sched_d, sse):
+        super(F8995, f).__init__(inputs)
+
+        if inputs['status'] == FilingStatus.JOINT:
+            f['2']  = sse[0]['3'] - sse[0]['13'] if sse[0].mustFile else 0
+            f['2'] += sse[1]['3'] - sse[1]['13'] if sse[1].mustFile else 0
+        else:
+            f['2']  = sse['3'] - sse['13'] if sse.mustFile else 0
```

Comment:
For line 2, how about something like this?
```
f['2'] = f1040['s1_3'] - (f1040.rowsum(['s1_1[567]']) or 0)
```
The rules around what counts as QBI are complicated, but I think this one captures at least as much as what you've got above.